posts/2021-11-03-tensorflow-gpu.rst

```diff
@@ -0,0 +1,55 @@
+.. post:: 2021-11-03
+   :tags: conda-forge
+   :author: wolfv
+
+.. role:: raw-html(raw)
+   :format: html
+
+GPU enabled TensorFlow builds on conda-forge
+============================================
+
+.. image:: https://i.imgur.com/pRdJaYw.png
+  :width: 400
+  :alt: Tensorflow on Anvil
+
+Recently we've been able to add GPU-enabled TensorFlow builds to conda-forge! This was quite a journey, with multiple contributors trying different ways to convince the Bazel-based build system of TensorFlow to build CUDA-enabled packages. But we managed, `and the pull request got merged <https://github.com/conda-forge/tensorflow-feedstock/pull/157>`_.
+
+We now have a configuration in place that creates CUDA-enabled TensorFlow builds for all conda-forge supported configurations (CUDA 10.2, 11.0, 11.1, and 11.2). Building out the CUDA packages requires beefy machines – on a 32 core machine it still takes around 3 hours to build a single package. Our build matrix now includes 12 CUDA-enabled packages & 3 CPU packages (because we need separate packages per Python version). As one can imagine, this isn't easily possible on an average "home computer".
```

Comment:
Might be worth noting that the 11.2 build should work with newer CUDA versions thanks to [CEC]( https://docs.nvidia.com/deploy/cuda-compatibility/index.html#default-to-minor-version ).

```suggestion
We now have a configuration in place that creates CUDA-enabled TensorFlow builds for all conda-forge supported configurations (CUDA 10.2, 11.0, 11.1, and 11.2+). Building out the CUDA packages requires beefy machines – on a 32 core machine it still takes around 3 hours to build a single package. Our build matrix now includes 12 CUDA-enabled packages & 3 CPU packages (because we need separate packages per Python version). As one can imagine, this isn't easily possible on an average "home computer".
```